control-plane/grpc/src/operations/replica/traits.rs

```diff
@@ -117,6 +118,7 @@ impl TryFrom<replica::Replica> for Replica {
                     ))
                 }
             },
+            allowed_hosts: vec![],
```

Comment:
Question: Why do we have different definition of proto and and transport type?

Reply:
what do you mean?

Reply:
Did not see this field in Replica structure.

Reply:
Okay. If its not needed then its fine.

Reply:
I've added it :)